contents/founders/what-recruiters-see.md

```diff
@@ -0,0 +1,77 @@
+
+As I sifted through these, I realized a very obvious thing - there is a _much_ bigger gap between what applicants and recruiter experience than either group realizes:
+
+- What applicants and recruiters see on their screens are _completely_ different. This leads to some needlessly funky looking applications - luckily this is easy to solve. 
+- You may spend 30min+ researching a company, crafting a cover letter and applying. At these numbers, I was able to spend **less than 1min** reviewing each application.  And this is just an average - some applications took me 5s(!) to realize they were a no, so I could spend 5min+ reading good ones carefully.
+
+Remember, actual professional recruiters don't just spend their time reviewing applications - they are conducting and scheduling interviews, collating feedback, negotiating offers, onboarding new hires, creating headcount plans, putting up job ads, etc. While it may seem extremely harsh that your application only gets a quick review, bear in mind that we're trying to figure out whether to invest dozens of people hours putting you through the full hiring process. 
+
+By sharing a bit of insight into how the _application_ bit of the hiring process actually works at a startup like PostHog, hopefully I can help increase your chances of landing a first round interview at least! Beyond that, no promises...
+
+## But first, some context
+
+I'm talking about startups with a reasonable degree of brand recognition (ie. not a startup founded last week). In PostHog's case, we are:
+
+- 42 people, $27m raised
+- Team is 70% engineers
+- 1 in-house recruiter, no agencies
+
+Let’s get into it. 
+
+## So what happens to my application after I hit send?
+
+If you're applying for a job at a startup, chances are they are using an applicant tracking system (ATS) like Ashby, Greenhouse, Lever, or Workable. We use Ashby, but the process a recruiter follows generally doesn't vary massively between these systems. This is what a typical application looks like:
+
+![Ashby screenshot](../images/ashby-screenshot.png)
+
+## How do I maximize my chances of getting through?
+
+This stuff is subjective, but generally these are the things that can increase the chances of a good application making it through:
+
+- Write a short and actually personalized proper cover note
+  - A few sentences about why you are interested in this role specifically is fine
+  - You don't need 5 paragraphs that are just your resumé slightly reworded - it's just more text we will skim over
+  - Our ATS will wreck your formatting beyond line breaks
+  - Don't introduce yourself (we know who you are)
+  - Match the tone of the company - for example, we are informal so please no 'Dear Sir/Madam'
```

Comment:
not sure this is exactly right, but an extra point around tailoring and being clear it's not mass spam being sent to us
```suggestion
  - Match the tone of the company - for example, we are informal so please no 'Dear Sir/Madam'
  - If you can figure out what the company cares about, or who is in it - talk about how you think you could help them achieve their goals and why
```